### DISCUSSION TITLE
Feature request: rendering of "boundary=protected_area"

### DISCUSSION THREAD

**Discussion by vitoller** [osmandapp/OsmAnd#19144] [Ideas]
I am using OSMAND for most of my tour planning.
Especially in winter there are a number of restricted areas (wildlife reserves), which should not be entered for the protection of wildlife.

Currently it is not easy to see these areas when planning a tour.

There is a online map (from the german alpine club) which shows the areas quite well, when opening the map. Also when clicking on the area  the kind and duration of the restriction is shown.

[https://www.alpenvereinaktiv.com/de/tour/von-der-schoenfeldalm-zur-rotwand-ueber-rauheck-und-taubenstein-am-2022-11-12/256473872/#dm=1](https://www.alpenvereinaktiv.com/de/tour/von-der-schoenfeldalm-zur-rotwand-ueber-rauheck-und-taubenstein-am-2022-11-12/256473872/#dm=1)

(I am not sure if I am allowed to post a screenshot from the website here)

**Comment by mariush444** [comment]
This renderer file has good visibility of NP. Maybe it help you.
AFAIK it is taken from Telegram osmand spain group
[rendering.zip](https://github.com/osmandapp/OsmAnd/files/14352603/rendering.zip)

**Comment by vitoller** [comment]
OK, so I did do some digging, and I believe what I am missing is the rendering of "**[boundary=protected_area](https://wiki.openstreetmap.org/wiki/Tag:boundary%3Dprotected_area)**" 

It seems that changing rendering styles is quite an art. Maybe someone can help me here (@sonora ?)

I would like to implement:

- outline of the areas (maybe yellow or red, depending on restrictions)
- the area filled but more transparent
- an icon for the type (maybe)
- when clicking on it with extra information maybe with
  - name
  - description
  - protection_title
  - seasonal
  - source

Any ideas/comments on this?

Where would I best start to implement the renderings?

Links:
[Digitize the Planet](https://content.digitizetheplanet.org/en/)
protected area examples: [In der Feile](https://content.digitizetheplanet.org/en/osm/tagging-suggestions/182008fb-a870-4f04-8ad1-bba961dd4911/)
[Core Area in the Bavarian Forest National Park](https://content.digitizetheplanet.org/en/rules/show_protectedarea/54f9d346-c94f-438a-98e3-4f6088a14b1a/)
